### ISSUE TITLE
Net measure regression in 4.5.1 when update interval is set to 100ms

### ISSUE THREAD

**Issue by Alkochm** [rainmeter/rainmeter#278]
In version 4.5.0 measure returns values every 100ms
In version 4.5.1 and higher measure returns values only once every ~300ms, other values are 0

Sample config:
[Rainmeter]
Update=100

[NET_IN_100ms]
Measure=NetIn
Interface=Best
DynamicVariables=1

**Comment by brianferguson** [comment]
This isn't really a regression since we have recently consolidated our network interface functions. Whenever there is a Net measure or SysInfo measure, the interface table is cached up to 4 times a second (or 250 milliseconds).

Here is the code for it:
[https://github.com/rainmeter/rainmeter/blob/master/Common/NetworkUtil.cpp#L23-L41](https://github.com/rainmeter/rainmeter/blob/master/Common/NetworkUtil.cpp#L23-L41)

Caching the value has a huge performance benefit, with the downside of not being able to "poll" the value super quickly. We figured 4 times a second would be plenty.

We will discuss this internally and possibly change the 250 millisecond interval if deemed necessary. I am going to close this issue for now. Thank you for bringing this to our attention.

-Brian
